docs/user/troubleshooting.md

```diff
@@ -50,9 +50,9 @@ sudo crictl pods --name=etcd-member
 
 If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
-### Unable to SSH into Master Nodes
+### Unable to access the Master Nodes via SSH 
 
-In order to SSH into the master nodes as user `core`, it is necessary to include an administrator's SSH key during the installation. When asked by the installation wizzard for the ssh-key, make sure you define the absolute path to your key. This key will be added to the core user's authorized_keys file. If installed in aws you will not see this key attached to your instance as a keypair in the aws console. 
+In order to ssh into the master nodes as user **core**, it is necessary to include an administrator's ssh-key during the installation. When asked by the installation wizard for the ssh-key, make sure you select your ssh-key from the wizard prompt **~/.ssh/*.pub**. This key will be added to the **core** user's **authorized_keys** file. If installed in AWS you will **not** see this key attached to your instance as a keypair in the AWS console. 
```

Comment:
This is moving in the wrong direction with `SSH` -> `ssh`.  Per [this comment][1], you should be using "SSH" (for the protocol) everywhere unless you are writing `ssh ...` for a shell command.

Also, `core` is more appropriate than **core**, because the username is literal [code][2] you could paste into shell scripts.  Similarly for your paths (`~/.ssh/*.pub` and `authorized_keys`).

*not* would be more appropriate than **not**, because this is for local [emphasis][3].  For example, see "[However, it should *not* be used...][4]" and "[This is *not* a drill][5]".  And [the AWS docs][6] use "key pair", not "keypair".  So I'd rather see something like:

> The public key is placed in `authorized_keys` by Ignition; it is not configured via platform-specific approaches like [AWS key pairs][6].

instead of your last two sentences.

[1]: https://github.com/openshift/installer/pull/795#discussion_r239547838
[2]: https://daringfireball.net/projects/markdown/syntax#code
[3]: https://daringfireball.net/projects/markdown/syntax#em
[4]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#Usage_notes
[5]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em
[6]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html